### PR TITLE
Add indexing speed improvement to release highlights

### PR DESCRIPTION
**Pull request by andreidan**
Describe https://github.com/elastic/elasticsearch/pull/63226 in the release highlights

### CODE REVIEW

File: docs/reference/release-notes/highlights.asciidoc [hunk 1]
```diff
@@ -25,6 +25,19 @@ Other versions:
 
 // tag::notable-highlights[]
 [discrete]
+[[indexing-speed-improvement]]
+===  Indexing speed improvement
+
+{es} 7.10 improves the indexing speed by 5-20%, depending on your workload.
+We've reduced the amount of coordination needed to add entries to the
+<<index-modules-translog,transaction log>>, enabling more concurrency, and
+incresed the transaction log buffer size from `8KB` to `1MB`.
```
Comment (andreidan) [inline docs/reference/release-notes/highlights.asciidoc:34]: is this too low level? (as it's not something the users can configure)

File: docs/reference/release-notes/highlights.asciidoc [hunk 2]
```diff
@@ -25,6 +25,19 @@ Other versions:
...
+
+The performance gains are lower in intensive analysis scenarios (ie. full-text
+search analysis).
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights.asciidoc:37]: I made some minor changes to improve the flow. However, the current text is
fine as-is as long as the `incresed` typo is fixed.
```suggestion
{es} 7.10 improves indexing speed by 5-20%. We've reduced the coordination
needed to add entries to the <<index-modules-translog,transaction log>>. This
reduction allows for more concurrency and increases the transaction log buffer
size from `8KB` to `1MB`.

These performance gains are lower for full-text search and other intensive
analysis scenarios.
```

File: docs/reference/release-notes/highlights.asciidoc (hunk 1 above)
Comment (jrodewig) [reply docs/reference/release-notes/highlights.asciidoc:34]: I think this is still worth highlighting as it's a change users may notice or be interested in.

**Review by jrodewig — APPROVED** [review]
LGTM. I left a suggestion, but feel free to leave as-is if wanted. I'd only request that we fix the `incresed` typo before merging.

Thanks!

File: docs/reference/release-notes/highlights.asciidoc [hunk 3]
```diff
@@ -25,6 +25,19 @@ Other versions:
 
 // tag::notable-highlights[]
 [discrete]
+[[indexing-speed-improvement]]
+===  Indexing speed improvement
+
+{es} 7.10 improves the indexing speed by 5-20%, depending on your workload.
```
Comment (jpountz) [inline docs/reference/release-notes/highlights.asciidoc:31]: Maybe say "up to 20%" without providing a lower bound. As you highlighted later, scenarios with full-text indexing or ingest might not notice a speedup.

File: docs/reference/release-notes/highlights.asciidoc (hunk 1 above)
Comment (jpountz) [inline docs/reference/release-notes/highlights.asciidoc:34]:
```suggestion
increased the transaction log buffer size from `8KB` to `1MB`.
```

File: docs/reference/release-notes/highlights.asciidoc (hunk 2 above)
Comment (jpountz) [reply docs/reference/release-notes/highlights.asciidoc:37]: +1 to James' suggestion. I'd mention ingest pipelines too. Something along the lines of "The heavier the indexing chain, the lower the gains, so indexing chains that involve many fields, ingest pipelines or full-text indexing will see lower gains."

**Review by jpountz — APPROVED** [review]

**Comment by andreidan** [comment]
@elasticmachine update branch

**Pull request merged**